src/snowflake/snowpark/row.py

```diff
@@ -14,6 +14,17 @@ def _restore_row_from_pickle(values, named_values, fields):
     return row
 
 
+def is_already_quoted(value: str) -> bool:
```

Comment:
there is a similar check present in `udf_utils.py` but importing it created circular import errors. is there a better way to do this?

Reply:
Maybe we could extract it into `util.py`?